blazor/mention/accessibility.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: Accessibility in Blazor Mention Component | Syncfusion
+description: Checkout and learn here all about accessibility in Syncfusion Blazor Mention component and much more details.  
+platform: Blazor
+control: Mention
+documentation: ug
+---
+
+# Accessibility in Blazor Mention Component
+
+The Mention component is designed to be compliant with `WAI-ARIA` (Web Accessibility Initiative - Accessible Rich Internet Applications) specifications, which provide guidelines and standards for making web content more accessible to people with disabilities. To achieve `WAI-ARIA` support, the Mention uses attributes such as `aria-selected` and `aria-activedescendent`. 
+
+The `aria-selected` attribute is used to indicate that an element is currently selected or has been selected in the past, while the `aria-activedescendent` attribute is used to indicate the currently active descendant element of a composite widget.
+
+## ARIA attributes
+
+The Mention component uses the listbox role and option role to implement the Mention functionality. The listbox role represents a list of options, and the option role represents a single item in the list. 
+
+It uses the various `ARIA attributes` (Accessible Rich Internet Applications) to denote the state of the component and provide additional information to assistive technologies such as screen readers. These attributes can help users who rely on assistive technologies to understand and interact with the Mention component more easily.
+
+Here are some of the `ARIA attributes` that might be used to denote the state of a mention list item:
+
+| **Properties** | **Functionalities** |
+| --- | --- |
+| `aria-selected` | Indicates the currently selected option in the list of mention suggestions |
+| `aria-activedescendent` | This attribute holds the ID of the active list item to focus its descendant child element. |
+| `aria-owns` | Indicate that the Mention component owns or controls the popup list of mention suggestions. |
+
+## Keyboard interaction
+
+You can use the following key shortcuts to access the Mention without interruptions. It allows users to quickly perform actions or navigate through an application using keyboard input.
+
+| **Keyboard shortcuts** | **Actions** |
+| --- | --- |
+
+| <kbd>Down arrow</kbd> | Focus the first item in the Mention list. Otherwise, focus the item next to the currently focused item. |
```

Comment:
The table not display properly.

<img width="710" alt="image" src="https://user-images.githubusercontent.com/29137312/209327738-4a7d49b2-83cf-4142-b372-87a88929830a.png">

Reply:
Addressed